COMET.Web.Common/ViewModels/Components/ParameterEditors/ParameterTypeEditorBaseViewModel.cs

```diff
@@ -58,13 +58,25 @@ public abstract class ParameterTypeEditorBaseViewModel<T> : DisposableObject, IP
         private ValueArray<string> valueArray;
 
         /// <summary>
-        /// Creates a new instance of type <see cref="ParameterTypeEditorBaseViewModel{T}" />
-        /// </summary>
-        /// <param name="parameterType">the parameter type of this view model</param>
-        /// <param name="valueSet">the value set asociated to this editor</param>
-        /// <param name="isReadOnly">The readonly state</param>
-        /// <param name="valueArrayIndex">the index of the value changed in the value sets</param>
-        protected ParameterTypeEditorBaseViewModel(T parameterType, IValueSet valueSet, bool isReadOnly, int valueArrayIndex = 0)
+        /// Gets the <see cref="ParameterSwitchKind" /> of the <see cref="CompoundParameterType"/>
+        /// </summary>
+        public ParameterSwitchKind CompoundCurrentParameterSwitchKind { get; set; }
+
+		/// <summary>

```

Comment:
Weird indentation here. Make sure that "Spaces"  is selected inside VS for this file